src/main/java/pl/krakow/controller/CustomerController.java

```diff
@@ -0,0 +1,70 @@
+package pl.krakow.controller;
```

Comment:
The naming could and basically should be more informative and precise. E.g. this class is an _API_ controller so _api_ would be a good part of  the package name. Also _pl.krakow_ says nothing about the application an its purpose.

Reply:
done!